recipe/meta.yaml

```
@@ -37,9 +37,10 @@ source:
     - patches/0002-don-t-use-C-20-stdlib-features-which-change-ABI-comp.patch
     # workaround for https://github.com/abseil/abseil-cpp/issues/1624
     - patches/0003-unconditionally-export-Mutex-Destructor.patch
+    - patches/0004-test-allocator.patch
```

Please add proper context and attribution for the patch
```suggestion
    # backport https://github.com/abseil/abseil-cpp/commit/779a3565ac6c5b69dd1ab9183e500a27633117d5
    - patches/<will-be-renamed,-see-below>.patch
```

---

updated as requested